scripts/testdata/satellite-config.yaml.lock

```diff
@@ -307,6 +307,9 @@ kademlia.operator.wallet: ""
 # number of update requests to process per transaction
 # overlay.update-stats-batch-size: 100
 
+# the length of time we will cache uptime status before updating the overlay cache again
+# overlay.uptime-flush-interval: 1m0s
```

Comment:
how did we decide on 1 min?

Reply:
Mostly as a guess, to be honest. I assumed that it would be fairly easy to change if we need to tweak it. The real question is how long we are ok going without updating reputation metrics. During this internal, if the uptime changes, we will not change the reputation of the node. So it depends on what interval we're comfortable waiting.